client/src/components/rooms/RoomsMeetingExperience.tsx

```diff
@@ -269,7 +275,9 @@ const RoomsMeetingExperience = (props: RoomsMeetingExperienceProps): JSX.Element
         }
         if (state?.call?.state === 'Connected') {
           setCallConnected(true);
-          setServerCallId(await state.call.info?.getServerCallId());
+          const serverCallId = await state.call.info?.getServerCallId();
+          setServerCallId(serverCallId);
+          updateParticipants({ displayName, userId: userId }, serverCallId);
```

Comment:
fn name is too generic, update what participants? why? where?
```suggestion
          sendParticipantInfoToServer({ displayName, userId }, serverCallId);
```

maybe add a comment also
```
          // The server needs to store the information of participants in the call for later mapping participantIds to display names in the transcription
          sendParticipantInfoToServer({ displayName, userId }, serverCallId);
```